src/openmc_nek_driver.cpp

```diff
@@ -346,45 +391,56 @@ void OpenmcNekDriver::update_temperature()
 
         // Set temperature for cell instance
         average_temp /= total_vol;
+        Expects(average_temp > 0.0)
```

Comment:
Not sure if you want `Expects` or `Ensures` here.

Reply:
`Expects` is fine. You do need a semicolon at the end of this though.

Reply:
`clang-tidy` actually complained that the semicolon is redundant, and I can safely omit it on both my machine and Travis.  But I looked at `gsl.h`, and it does seem like it's necessary in general:  

``` cpp
#if gsl_ELIDE_CONTRACT_EXPECTS
# define Expects( x )  /* Expects elided */
#elif gsl_CONFIG( CONTRACT_VIOLATION_THROWS_V )
# define Expects( x )  ::gsl::fail_fast_assert( (x), "GSL: Precondition failure at " __FILE__ ":" gsl_STRINGIFY(__LINE__) );
#else
# define Expects( x )  ::gsl::fail_fast_assert( (x) )
#endif
```
One of the macro definitions has a semicolon at the end, but the other doesn't.  They need to be consistent...

Anyway, I'll add the semicolon.